package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "alpheios-lexicon-client",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "Alpheios Lexicon Client Library",
-  "main": "dist/alpheios-lexicon-client.standalone.js",
-  "moduleExternal": "dist/alpheios-lexicon-client.module-external.js",
+  "main": "dist/alpheios-lexicon-client.js",
   "scripts": {
     "test": "standard --fix && jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "build": "standard --fix && node ./rollup.build.js",
+    "build": "standard --fix && node --experimental-modules ./node_modules/alpheios-node-build/dist/build.mjs all",
```

Comment:
To match the latest `node-build` it needs to have all three build parameters listed, i.e.
`build.mjs all all lib` for `build`, `all production lib` for `prod`, and `all development lib` for dev.